shared/src/business/useCases/judgeActivityReport/getOpinionsFiledByJudgeInteractor.ts

```diff
@@ -22,43 +23,55 @@ import { orderBy } from 'lodash';
  * @returns {array} list of opinions filed by the judge in the given date range, sorted alphabetically ascending by event code
  */
 export const getOpinionsFiledByJudgeInteractor = async (
-  applicationContext,
-  {
-    endDate,
-    judgeName,
-    startDate,
-  }: { judgeName: string; endDate: string; startDate: string },
-) => {
+  applicationContext: object,
+  params: JudgeActivityReportFilters,
+): Array<any> => {
```

Comment:
Looks like this function doesn't return anything now, can the return type be changed to reflect that?

Reply:
Good catch. This has been resolved in this (PR)[https://github.com/ustaxcourt/ef-cms/pull/3841]